apps/www/code-hike.d.ts

```diff
@@ -0,0 +1,6 @@
+import { remarkCodeHike } from '@code-hike/mdx'
+
+declare module '@code-hike/mdx' {
+  export type CodeHikeRemarkPlugin = typeof remarkCodeHike
+  export type CodeHikeConfig = Parameters<CodeHikeRemarkPlugin>[0]
+}
```

Comment:
Could this also be moved to a shared config?

Reply:
Ideally yeah, but we can wait to do this with other type files too that should also be shared